README.md

```diff
@@ -20,7 +20,7 @@ Our template offers a rich boilerplate to jump-start React-based application dev
 
 ### Prerequisites
 
-[![node-version-image][node-v-src]](https://nodejs.org/download/release/v14.18.1/)
+[![node-version-image][node-v-src]](https://nodejs.org/download/release/v14.18.2/)
```

Comment:
Should we change to use the same version as we set on `.nvmrc` file `16.14.2`? That is the LTS version, then I think we can switch to use it from now

Reply:
@hoangmirs  do you have an idea how to change the displayed Node Version? Because at first I've tried to update the version, but it's only the link that got updated, not the image displaying the node version...

Reply:
You can see in 3661706, this change results in:

<img width="470" alt="image" src="https://user-images.githubusercontent.com/77609814/159434769-a9016d0b-f23c-4522-85f9-4a50cca18e59.png">

Reply:
(or will that be updated once we would release a new version of the template?)

Reply:
Fixed! 252365a ✅ 

I removed the variable to the node image because it makes no sense to move this at the bottom of the README file when we need to update the link at the same time... 🤔